pkg/apis/pravega/v1alpha1/register.go

```diff
@@ -8,39 +8,20 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
 
+// Package v1alpha1 contains API Schema definitions for the pravega v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
```

Comment:
Remove commented out lines if they aren't needed.

Reply:
This file is generated by the SDK. The `+k8s:...` line is used by the SDK to keep track of generated files. Ref: https://github.com/operator-framework/operator-sdk/blob/00ef545399db0e4b8fe4474188a342369874162a/doc/sdk-cli-reference.md#generate